yomigeek/mtracker-app
# #158248400 Implement Admin Single Request Interface                                                                                                                                                                                                                                                                 ft(admin-single-request-ui): Implement Admin Single Request

#### What does this PR do?
Implement Admin Single Request Interface

#### Description of Task to be completed?
Admin should be able to view a single request details

#### How should this be manually tested?
Go to the url https://mtrack-app.herokuapp.com/login.html
Enter the required credentials
You get a response if its successful or not
If successful the user dashboard page is displayed 
Admin click View for any request that is required to view it's details 

#### Any background context you want to provide?
None

####  What are the relevant pivotal tracker stories?
story type: feature
story id: 158248400

#### What are the relevant Github Issues?
None

#### Questions:
None
